core/src/components/range/range.md.scss

```diff
@@ -63,6 +64,28 @@
   transition-timing-function: ease;
 
   z-index: 2;
+
+  &::before {
+    @include border-radius(50%);
+    position: absolute;
+    top: -9px;
+    /* stylelint-disable property-disallowed-list */
+    left: -9px;
+    /* stylelint-enable property-disallowed-list */
+
+    width: 36px;
+    height: 36px;
+
+    transform: scale(0.1);
```

Comment:
This also comes from Vuetify, but the intention is that when the indicator begins to scale to full size, it will start at a scale value closer to the existing knob size. This creates a more "smooth" growth of indicator behind the knob.

Reply:
When not make the focus indicator the same size as the knob? Then you can have it transform from `scale(1)` to `scale(>1)`? This is what the web implementation for Material Design components do: https://material-components.github.io/material-components-web-catalog/#/component/slider. That way you don't have to guess at an approximate size.